opendocdb-cts/main.go

```diff
@@ -124,6 +126,23 @@ func convertCommand() error {
 	return nil
 }
 
+// prepareOutDir creates subdirectory inside the output directory specified by flag.
+// It returns the created path if succeeded.
+func prepareOutDir(relpath ...string) (string, error) {
```

Comment:
It looks weird that we both kept a function for calling _one_ other function _and_ increated cognitive-complexity's parameter

Reply:
This specific change actually doesn't change cognitive complexity. In both cases (with and without this function) we need to do `if` error check.

I extracted it to separate function, as we repeat the same code for 3 different directories, although the change is unnecessary, so I'll revert it.